tests/mock_module/api.py

```diff
@@ -19,7 +19,7 @@ class Record(RecordBase):
     schema = ConstantField(
         '$schema', 'http://localhost/schemas/records/record-v1.0.0.json')
 
-    index = IndexField('records-record-v1.0.0', search_alias='records')
+    index = IndexField('records-record-v1.0.0', search_alias='draftsresources-records')
```

Comment:
Question: what's the reason for this change?

Reply:
@lnielsen The agreement we got to have the folders like:

```
draftsresources/    # equivalent to rdmrecords
    records/
        ...
    drafts/
        ...
```

To allow index/alias creation. How `invenio-search` works is that it will create:
- `draftsresources`: queries all indices
- `draftsresources -records`: queries all indices inside that folder
- `draftsresources -drafts`: same as above but for drafts

meaning it prefixes all sort of in a recursive way.

Reply:
Ahh, got it :-) Slow today.... 😄